.github/workflows/ci.yml

```diff
@@ -217,13 +221,19 @@ jobs:
         with:
           # We need to fetch all commits so that we can diff against the base branch
           fetch-depth: 0
+
+      - name: Setup source policy
+        uses: ./.github/actions/setup-source-policy
       - name: Expose GitHub tokens for caching
         uses: crazy-max/ghaction-github-runtime@b3a9207c0e1ef41f4cf215303c976869d0c2c1c4 # v3.0.0
+      - name: Setup source policy
```

Comment:
Should it be duplicated here?

Reply:
it is not duplicated but rather called once in the integration test job and e2e test job, two different runners

Reply:
I think it was duplicated and I fixed it :)